AdminWebsite/AdminWebsite/ClientApp/src/app/booking/judicial-office-holders/models/add-judicial-member.model.ts

```diff
@@ -31,6 +33,9 @@ export class JudicialMemberDto {
         dto.optionalContactEmail = response.optional_contact_email;
         dto.roleCode = response.role_code as JudicaryRoleCode;
         dto.displayName = response.display_name;
+
+        // todo: map from response when API is implemented
```

Comment:
there's a TODO here, is that part of a separate ticket?

Reply:
Yup, there's a ticket for the API changes that need to update the contracts too